src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -1241,7 +1241,7 @@ public static implicit operator Vector4(Color color)
             {
                 return null;
             }
-            return new Vector4(color.R, color.G, color.B, color.A);
+            return ObjectPool.NewVector4(color.R, color.G, color.B, color.A);
```

Comment:
I think we cannot assume that `implicit operator Vector4(Color color)` API called only at main thread.

ObjectPool is not thread safe class. So we MUST not change like this.

Reply:
Done.